### PR TITLE
Update history-mode.md

### PR DESCRIPTION
**Pull request by laoer536**
I think use this word is better. Thanks.

### CODE REVIEW

**Review by sxzz — APPROVED** [review]

**Review by Jinjiang — APPROVED** [review]

File: packages/docs/zh/guide/essentials/history-mode.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ const router = createRouter({
 
 当使用这种历史模式时，URL 会看起来很 "正常"，例如 `https://example.com/user/id`。漂亮!
 
-不过，问题来了。由于我们的应用是一个单页的客户端应用，如果没有适当的服务器配置，用户在浏览器中直接访问 `https://example.com/user/id`，就会得到一个 404 错误。这就丑了。
+不过，问题来了。由于我们的应用是一个单页的客户端应用，如果没有适当的服务器配置，用户在浏览器中直接访问 `https://example.com/user/id`，就会得到一个 404 错误。这就糗了。
```
Comment (sxzz) [inline packages/docs/zh/guide/essentials/history-mode.md:44]: How about change it to `尴尬`?

Comment (laoer536) [reply packages/docs/zh/guide/essentials/history-mode.md:44]: OK

**Comment by posva** [comment]
Thanks! I merged #1606 with 尴尬

**Comment by laoer536** [comment]
> 谢谢！我合并#[1606](https://github.com/vuejs/router/pull/1606)与尴尬
OK! Thanks.

**Pull request closed without merging**